mattermost-plugin/webapp/src/index.tsx

```diff
@@ -140,15 +139,14 @@ export default class Plugin {
                 window.open(`${windowAny.frontendBaseURL}/workspace/${currentChannel}`)
             }
             this.channelHeaderButtonId = registry.registerChannelHeaderButtonAction(<FocalboardIcon/>, goToFocalboardWorkspace, '', 'Boards')
+            this.registry.registerProduct('/boards', 'product-boards', 'Boards', '/plug/focalboard/go-to-current-workspace', MainApp, HeaderComponent)
 
             this.registry.registerCustomRoute('go-to-current-workspace', () => {
-                const history = useHistory()
                 useEffect(() => {
-                    history.replace(`${windowAny.frontendBaseURL}/dashboard`)
+                    window.location.replace(`${windowAny.frontendBaseURL}/welcome`)
```

Comment:
@jespino 

For some reason this kept redirecting me to a session expired when `lastWorkspaceViewed` in localstorage was deleted. I decided to use window.location.replace which works however it does a full refresh.

I'm not sure why. Hoping you can maybe have an input on this

Reply:
Actually, all this code is no longer needed, this was a solution to redirect you to the current workspace, but now, you are no longer getting redirected to the current workspace, so... we only need to set the right url in the `registerProduct` call. Something like `/boards/welcome` instead of `/plug/focalboard/go-to-current-workspace`, and you can remove the entire `this.registry.registerCustomRoute` there.

Reply:
Thanks! Done.